cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -114,8 +114,16 @@ export default {
   },
   methods: {
     ...mapGetters('agent', { getAgentInstance: 'getInstance' }),
-    ...mapGetters(['getCurrentUser', 'getAgentOpts']),
+    ...mapGetters(['getCurrentUser', 'getAgentOpts', 'getStaticAssetsConfigData']),
```

Comment:
`getStaticAssetsConfigData` name is too generic and ambiguous.

Reply:
updated